crate_universe/src/rendering/templates/partials/crate/aliases.j2

```diff
@@ -1,9 +1,9 @@
 selects.with_or({
     {%- for cfg, values in selectable.selects %}
-    {%- if cfg in context.conditions and context.conditions[cfg] | length %}
+    {%- if context.conditions | get(key=cfg, default="") | length%}
```

Comment:
The changes here are to account for the change in behavior from https://github.com/Keats/tera/issues/738